code/_onclick/ai.dm

```diff
@@ -63,6 +66,10 @@
 		waypoint_mode = 0
 		return
 
+	if(controlled_mech) //Are we piloting a mech? Placed here so the modifiers are not overridden.
```

Comment:
Okay, how about this, @Aranclanos  dislikes that every click checks for the AI piloting a mech. Well, I can simply remove that check, and reverse the functions so that the AI player must middle click to activate the mech functions, and normal clicks are entirely unaffected! Middle click *has no function for AI*, so the only reason an AI player would use it is for a mech! 

No need to check every click for a mech, middle click will still allow the AI to work in a mech, and AI does not lose its AI abilities while in a mech, it will only require a slight adjustment in behalf of the player, of which the player is informed in green text (the most attractive to player eyes) how to operate the mech as an AI! Is this solution acceptable?